js/index.js

```diff
@@ -137,3 +138,115 @@ function creationOwlCarousel () {
             e.preventDefault();
     });
 };
+
+/**Coding section for Cards */
+if (document.readyState == 'loading') {
+    document.addEventListener('DOMContentLoaded', ready)
+} else {
+    ready();
+}
+
+function ready() {
+    let removerCartItemButtons = document.getElementsByClassName('btn-danger');
+    //console.log(removerCartItemButtons);
+    for (let i = 0; i < removerCartItemButtons.length; i++){
+        let button = removerCartItemButtons[i];
```

Comment:
you can simply write it in one line:
`removerCartItemButtons[i].addEventListener('click', removeCartItem);`
It's always good to write shorter code when possible.
And check below as well.